cardano-cli/src/Cardano/CLI/Commands/Governance.hs

```diff
@@ -61,6 +64,61 @@ data GovernanceCmdError
   | ShelleyGovernanceCmdGenesisDelegationNotSupportedInConway
   deriving Show
 
+instance Error GovernanceCmdError where
+  displayError = \case
+    StakeCredGovCmdError stakeAddressCmdError ->
+      "Stake credential error: " <> toS (renderShelleyStakeAddressCmdError stakeAddressCmdError)
+    VotingCredentialDecodeGovCmdEror decoderError ->
+      "Could not decode voting credential: " <> renderDecoderError decoderError
+    WriteFileError fileError ->
+      "Cannot write file: " <> displayError fileError
+    ReadFileError fileError ->
+      "Cannot read file: " <> displayError fileError
+    ExpectedStakeKeyCredentialGovCmdError ->
```

Comment:
We can actually delete `ExpectedStakeKeyCredentialGovCmdError`. The function that returns it isn't being used anywhere.